WordPress/src/main/res/values/strings.xml

```diff
@@ -1301,6 +1301,11 @@
     <string name="delete_site_summary">This action can not be undone. Deleting your site will remove all content, contributors, and domains from the site.</string>
     <string name="delete_site_hint">Delete your site</string>
     <string name="pref_key_site_start_over_screen">wp_pref_key_site_start_over_screen</string>
-    <string name="delete_site_progress">Deleting site...</string>
+    <string name="delete_site_progress">Deleting site…</string>
+    <string name="purchases_request_error">Something went wrong. Please contact support</string>
```

Comment:
We shouldn't say "Please contact support" without providing an obvious, immediate way to do so. In this case I think the feature isn't going to be widely used, and the error isn't very likely, so a generic error message without the support mention should suffice.